controller/space_areas_blackbox_test.go

```diff
@@ -86,8 +87,10 @@ func (rest *TestSpaceAreaREST) setupAreas() (area.Area, []uuid.UUID, []area.Area
 	*/
 	var createdAreas []area.Area
 	var createdAreaUuids []uuid.UUID
-	sp, parentArea := createSpaceAndArea(rest.T(), rest.db)
-	createdAreas = append(createdAreas, parentArea)
+	fxt := tf.NewTestFixture(rest.T(), rest.DB, tf.Spaces(1), tf.Areas(1))
```

Comment:
I think `createSpaceAndArea` do create a space with a root iteration, right? That's why I would simply go with:

```go
fxt := tf.NewTestFixture(rest.T(), rest.DB,
    tf.CreateWorkItemEnvironment(), 
    tf.Areas(4, func(fxt *tf.TestFixture, idx int) error {
        if idx > 0 {
            fxt.Areas[idx].MakeChildOf(*fxt.Areas[idx-1])
        }
    }),
)
```

This sets up a space with a root iteration and root area and it creates 3 additional areas where each is a child of the area before. This should render the `setupAreas` function obsolete as well.

Reply:
Please see https://github.com/fabric8-services/fabric8-wit/pull/1941/commits/8eb02d4505a090c9a189fe80fbcdd9ab53d9cb13